src/assets/sass/protocol/base/elements/_sections.scss

```diff
@@ -23,6 +23,28 @@
     }
 }
 
+.mzp-s-content {
```

Comment:
Rather than duplicating the entire declaration I think it would be more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) to make the narrow content a theme variation of the existing `mzp-l-content` class.

If people declared it as `<div class="mzp-l-content mzp-t-narrow">`. The only thing you would have to over-ride is the `max-width` declared on line 12 :)